src/pilot/config_idrac.py

```diff
@@ -147,7 +147,7 @@ def configure_uefi_nics_boot_settings(drac_client, pxe_nic_id):
                "PxeDev1VlanEnDis": "Disabled"
                }
 
-            provisioning_mac = nic.mac.lower()
+            provisioning_mac = nic.mac_address.lower()
```

Comment:
I don't think the above change is correct.  The current master python-dracclient has mac as an attribute for NICs, not mac_address: https://github.com/openstack/python-dracclient/blob/master/dracclient/resources/inventory.py#L179